modules/govuk/manifests/apps/mapit.pp

```diff
@@ -58,6 +58,9 @@
   }
 
   govuk::app::envvar {
+    "${title}-DB_PASSWORD":
+        varname => 'DB_PASSWORD',
```

Comment:
This would need to be `MAPIT_DB_PASS` to match up with your mapit change [1], is there something I'm missing?

1: https://github.com/alphagov/mapit/pull/42/commits/995c7d1d4478e450c6a675fc298094436710df15#diff-9ca8db372ff7b61122729b2518307c66R86

Reply:
You're right of course :-). I'll change this one, thanks

Reply:
Looks good 👍